tests/Curl/CurlTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\Std\Tests\Curl;
+
+use Brick\Std\Curl\Curl;
+use PHPUnit\Framework\TestCase;
+
+class CurlTest extends TestCase
+{
+    public function testConstructorShouldInitialCurl()
+    {
+        $curl = new Curl();
+
+        $this->assertInternalType('object', $curl);
```

Comment:
This test is not required, `new` always returns an object or throws an exception.

Reply:
I will remove this test.